HW06.Rmd

```diff
@@ -19,14 +20,27 @@ Steps:
 
 From the RNA-Seq files, you only need the "Name" from one file and the "TPM" column from all the files. TPM stands for "transcripts per million" and is a common unit for normalized expression data. 
 ```{r load in data}
-#hint, using apply (specifically sapply) you can read in the data into a list and then bind the columns together. Or you can use a for loop too. 
 
-#you only need the 
+#Load R packages
+library(readr)
+library(dplyr)
+library(tidyverse)
+
+#Create a list of files within folder "RNA_Seq_processed"
+rna_file.list <- list.files("D:/BSDG_30901/Project/HW06/RNA_Seq_processed", recursive = TRUE, full.names = TRUE)
+
+#Use lapply to read files into a data frame list 
+rna_df.list <- sapply(rna_file.list, read_tsv, simplify = FALSE)
+

```

Comment:
the way you read the files are very clear and efficient! I didn't that you can read files in two folders in the same directory, learned from your code about how.